docs/policies/policies.md

```diff
@@ -0,0 +1,480 @@
+
+        curl -X POST "${BOB_DATAMGMT_URL}/data/contractdefinitions" \
+            --header 'X-Api-Key: password' \
+            --header 'Content-Type: application/json' \
+            --data '{
+                        "id": "1",
+                        "criteria": [
+                            {
+                                "operandLeft": "asset:prop:id",
+                                "operator": "=",
+                                "operandRight": "1"
+                            }
+                        ],
+                        "accessPolicyId": "1",
+                        "contractPolicyId": "1"
+                    }' \
+            -s -o /dev/null -w 'Response Code: %{http_code}\n'
+
+As you can see, the contract definition determines which access or usage policy is referred to. This is done via the respective id's.
+
+>ℹ️ More detailed information about the data transfer in Catena-X can be found [here](https://github.com/eclipse-tractusx/tractusx-edc/blob/main/docs/samples/Transfer%20Data.md).
+
+### EDC Policy Playground
+>
+>ℹ️ Policies can be created individually with the following tool:
+
+- Online Version: [Link](https://eclipse-tractusx.github.io/tutorial-resources/policy-playground/)
+- Local Version: [Link](https://github.com/eclipse-tractusx/tutorial-resources/tree/main/edc-policy-playground)
+>
+### Exemplary scenarios
+
+For the following Scenarios, we assume there is a **Partner 1 (provider)** who wants to provide Data for **Partner 2 (consumer)**.
+
+- Partner 1 (provider) has the Business-Partner-Number BPN12345.
+- Partner 2 (consumer) has the Business-Partner-Number BPN6789.
+
+Partner 2 (consumer) signed the **Traceability Framework Agreement** and followed all the necessary steps that the Credential appears within Partner 2s identity.
+
+So for this example, if Partner 2 does a catalogue request to Partner 1, the following summary credential is provided to Partner 1:
+
+    {
+        "@context": [
+            "https://w3id.org/2023/catenax/credentials/summary/v1"
+        ],
+        "id": "<credential uid>",
+        "type": [
+            "VerifiableCredential",
+            "SummaryCredential"
+        ],
+        "issuer": "<did:web:issuer>",
+        "issuanceDate": "2023-06-02T12:00:00Z",
+        "expirationDate": "2022-06-16T18:56:59Z",
+        "credentialSubject": {
+            "id": "<did:web:subject>",
+            "holderIdentifier": "BPN6789",
+            "items": [
+            "MembershipCredential",
+            "TraceabilityCredential"
+            ],
+            "contractTemplates": "https://public.catena-x.org/contracts/"
+        }
+    }
+
+#### Scenario 1
+
+Partner 1 wants to create an Access Policy, that Partner 2 can only receive the Contract offer if its BPN matches AND if it has the Traceability Credential (Traceability Contract Signed). If one of those is missing, Partner 2 won't receive a Contract Offer.
+
+##### Partner 1 - Access Policy Example
+
+    {

```

Comment:
Code fences and indentation alignment please :-)

Reply:
done